### DISCUSSION TITLE
How to apply the style changes to the ID instead of the class?

### DISCUSSION THREAD

**Discussion by LamaYouzghatli** [GrapesJS/grapesjs#3804] [Q&A] [answered]
Hi,
I have two elements with the same classes. I want to apply style changes with style manager to one of these two elements. I add ID but the style applies to the class not to the ID.

What can I do?
I want to solve it from the code ..

**Comment by artf** [comment, marked as answer]
```js
grapesjs.init({
	...
	selectorManager: {
        componentFirst: true,
    },
})
```

**Comment by LamaYouzghatli** [comment]
Thanks a lot.
